source/_integrations/tankerkoenig.markdown

```diff
@@ -34,6 +34,7 @@ tankerkoenig:
   radius: 1
   fuel_types:
     - "diesel"
+  show_on_map: true
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

```suggestion
```